tests/TestCase/Command/RoutesCommandTest.php

```diff
@@ -71,26 +71,46 @@ public function testRouteListHelp()
      */
     public function testRouteList()
     {
-        $this->exec('routes');
+        $this->exec('routes -v -s');
```

Comment:
There should be test for `routes` too without any additional options.

Reply:
I think this meets near full coverage as-is. I could add a smaller test for just `-v -s` to test order and the additional column if you think that's better.

Reply:
@ADmad Is there an easy way to test the order of the output when using `--sort`?

Reply:
Cuz you ain't gonna like what I'm cooking up buddy.

Reply:
I just pulled something out of the oven for you buddy. Bon Appétit!